js/changeColor.js

```diff
@@ -0,0 +1,63 @@
+import {storage} from './main.js'
+
+let btn = document.getElementById('change-color-btn');
+
+/* Need this since the color picker wants hex 
+** to set the default value and apparently js
+** just convert hex to rgb when I put in a hex
+** value into the backgroundColor
+*/ 
+function RGBToHex(rgb) {
+    // Choose correct separator
+    let sep = rgb.indexOf(",") > -1 ? ", " : " ";
+    // Turn "rgb(r,g,b)" into [r,g,b]
+    rgb = rgb.substr(4).split(")")[0].split(sep);
+    
+    let r = (+rgb[0]).toString(16),
+    g = (+rgb[1]).toString(16),
+    b = (+rgb[2]).toString(16);
+    
+    if (r.length == 1)
```

Comment:
Can't help but feel that this is a bit redundant? it is sooo similar

Reply:
Do you mean the if statements? I guess you have to check every value to see if you need to add a zero or not